contracts/registry/TestRegistrar.sol

```diff
@@ -7,11 +7,11 @@ import "./ENS.sol";
  * expires registrations a fixed period after they're initially claimed.
  */
 contract TestRegistrar {
-    uint constant registrationPeriod = 4 weeks;
+    uint256 constant registrationPeriod = 4 weeks;
 
     ENS public ens;
```

Comment:
this can be immutable

Reply:
i'm not sure we care about the test registrar, but i'll add it anyway

Reply:
Edit, I just realised this is a constant, which is just as good as immutable since it is never set, but hardcoded